backend/data/config.py

```diff
@@ -15,7 +15,7 @@
 from typing import List
 
 
-LIVE_YEAR: int = 2022
+LIVE_YEAR: int = 2023
```

Comment:
I remember struggling to decide when to change the LIVE_YEAR (like after T3 census date? Before T3 census date? Before T3 Week 1? etc).

I think this issue is now resolved by the `terms_offered()` function in the routers so FE can still get accurate data for 22T3 for their term planner right?